content/news/028/index.md

```diff
@@ -31,6 +31,17 @@ Feel free to send PRs about your own projects!
 [join]: https://github.com/rust-gamedev/wg#join-the-fun
 
 - [Game Updates](#game-updates)
+
+### Country Slice
+
+![country-slice-gif](country_slice_gif.gif)
+
+[Country Slice](https://github.com/anopara/country-slice) is [@anastasiaopara](https://twitter.com/anastasiaopara/)'s hobby project, where users can draw a small scene, and their input is amplified with real-time procedural geometry generation.
+
+Country Slice uses [Bevy Engine](https://github.com/bevyengine/bevy) for entity management, and has recently [been ported to OpenGL](https://twitter.com/anastasiaopara/status/1464304076074672144?s=20) (using [gl-rs](https://github.com/brendanzab/gl-rs/tree/master/gl) and [glutin](https://github.com/rust-windowing/glutin)). It is being developed openly on [GitHub](https://github.com/anopara/country-slice). 
```

Comment:
For any links you have, can you extract them to the bottom?

Example: https://github.com/rust-gamedev/rust-gamedev.github.io/pull/860/files